OnlineShopping.rb

```diff
@@ -1,33 +0,0 @@
-=begin
```

Comment:
so from the dif changes looks like this file is already in master and you are deleting this file in this branch? that is the change you are making. so if this PR was merged, then this file will be deleted in master then